frontend/src/component/ui/ChangeLogModal.scss

```diff
@@ -3,113 +3,98 @@
-      padding: 20px;
-      height: 32px;
-      border-bottom: 1px solid $color-divider;
-      font-weight: 600;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    font-size: 20px;
+    line-height: 1;
+    text-align: left;
+    padding: 20px;
+    height: 32px;
+    border-bottom: 1px solid $color-divider;
+    font-weight: 600;
 
-      i.clear {
-        cursor: pointer;
-      }
+    i.clear {
+      cursor: pointer;
     }
+  }
 
-    .changelog {
+  .changelog {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    background-color: $color-modal-background;
+    overflow: auto;
+
+    ul {
       flex: 1;
-      display: flex;
-      flex-direction: column;
-      background-color: $color-modal-background;
-      overflow: auto;
+      list-style: none;
+      display: block;
+      padding: 0;
+      margin: 0;
+      overflow-y: scroll;
+      margin: 4px;
+      @include scroll-bar;
 
-      ul {
-        flex: 1;
+      li {
+        display: flex;
+        flex-direction: column;
+        padding: 10px;
         list-style: none;
-        display: block;
-        padding: 0;
-        margin: 0;
-        overflow-y: scroll;
-        margin: 4px;
-        @include scroll-bar;
-
-        li {
-          display: flex;
-          flex-direction: column;
-          padding: 10px;
-          list-style: none;
-          text-align: left;
-          border: 1px solid $color-divider;
-          margin: 10px;
-          min-height: 120px;
-          background: white;
-          word-break: break-word;
-
-          .title {
-            font-weight: 900;
-            margin-bottom: 6px;
-          }
-
-          .summary {
-            flex: 1;
-          }
+        text-align: left;
+        border: 1px solid $color-divider;
+        margin: 10px;
+        min-height: 120px;
+        background: white;
+        word-break: break-word;
 
-          .released-date {
-            color: $color-time;
-          }
+        .title {
+          font-weight: 900;
+          margin-bottom: 6px;
         }
-      }
-    }
 
-    .view-all-updates {
-      width: 100%;
-      height: 50px;
-      background-color: $color-modal-background;
+        .summary {
+          flex: 1;
+        }
 
-      button {
-        background-color: $color-divider;
-        color: $color-dark-gray;
-        height: 30px;
-        width: calc(100% - 24px);
-        margin: 10px;
-        margin-right: 20px;
+        .released-date {
+          color: $color-time;
+        }
       }
     }
   }
 
-  @media only screen and (max-device-width: 480px) {
-    .modal-body {
-      width: 100vw;
-      height: 100vh;
+  .view-all-updates {
+    width: 100%;
+    height: 50px;
+    background-color: $color-modal-background;
+
+    button {
+      background-color: $color-divider;
+      color: $color-dark-gray;
+      height: 30px;
+      width: calc(100% - 24px);
+      margin: 10px;
+      margin-right: 20px;
```

Comment:
Is this for the scrollbar?

Reply:
yes

Reply:
@kapillamba4 Let's for now remove margin-right. If look very wired, we can add it back.